posts/month-in-mathlib-may-2024.md

```diff
@@ -0,0 +1,89 @@
+    * [PR #11585](https://github.com/leanprover-community/mathlib4/pull/11585) proving that the category of light profinite objects is precoherent.
+* Number Theory.
+  * [PR #12897](https://github.com/leanprover-community/mathlib4/pull/12897) defines the **Hurwitz zeta function**, and proves its key properties (analytic continuation + functional equation). This is a generalisation of the Riemann zeta function, and is an important step towards Dirichlet L-functions in the near future. (Subsidiary PR's include [PR #12779](https://github.com/leanprover-community/mathlib4/pull/12779), [PR #12265](https://github.com/leanprover-community/mathlib4/pull/12265), [PR #13347](https://github.com/leanprover-community/mathlib4/pull/13347), [PR #13273](https://github.com/leanprover-community/mathlib4/pull/13273).)
+
+* Order Theory.
+  * Stone duality between the categories of Boolean algebras and of profinite spaces is slowly moving from the [StoneDualityInLean project](https://github.com/samvang/StoneDualityInLean/): [PR #12705](https://github.com/leanprover-community/mathlib4/pull/12705) proves that a disjoint filter and ideal in a bounded distributive lattice can always be separated by a prime ideal.
+  * Christopher Hoskin continues introducing topologies relevant to order theory: [PR #11710](https://github.com/leanprover-community/mathlib4/pull/11710) defines the Lawson Topology.
+
+* Metaprogramming.
+  * Miyahara Kō heroically ported the `cc` tactic to Lean 4: [PR #11956](https://github.com/leanprover-community/mathlib4/pull/11956), [PR #11960](https://github.com/leanprover-community/mathlib4/pull/11960) and [PR #5938](https://github.com/leanprover-community/mathlib4/pull/5938).
+  * Damiano Testa and Yaël Dillies replaced the `mk_all` shell script hardcoded to Mathlib with a Lean script that dynamically figures out the Lean libraries (sadly, Mathlib still needs some separate handling) in [PR #11853](https://github.com/leanprover-community/mathlib4/pull/11853), and [PR #11874](https://github.com/leanprover-community/mathlib4/pull/11874) made use of it in the "Check all files are imported" CI step. The script can now be used by any project downstream of Mathlib. If you maintain such a project, we encourage you to switch over to the new script and report eventual bugs.
+  * Vasily Nesterov implemented a simplex algorithm oracle in [PR #12014](https://github.com/leanprover-community/mathlib4/pull/12014). This is now the default oracle for `linarith` instead of the much slower Fourier-Motzkin oracle.
+  * [PR #13190](https://github.com/leanprover-community/mathlib4/pull/13190) and [PR #13293](https://github.com/leanprover-community/mathlib4/pull/13293) by Michael Rothgang and Damiano Testa neuters a very pernicious footgun: Contrary to expectations, `attribute [non_local_attribute] decl in` would **globally** tag `decl` with attribute `non_local_attribute`.
```

Comment:
```suggestion
  * [PR #13190](https://github.com/leanprover-community/mathlib4/pull/13190) and [PR #13293](https://github.com/leanprover-community/mathlib4/pull/13293) by Michael Rothgang and Damiano Testa neuters a very pernicious footgun: contrary to expectations, `attribute [non_local_attribute] decl in` would **globally** tag `decl` with attribute `non_local_attribute`.
```
maybe?  Although I see that the post has several occurrences of a capital letter after a colon, so maybe this is intended.

Reply:
I think I _slightly_ prefer capitalising after a colon since it is technically a new sentence starting